.github/workflows/set-milestone-on-referenced-issue.yml

```diff
@@ -0,0 +1,33 @@
+#############################################################################
+# GitHub action to set latest milestone on issue of merged PR.
+#
+#############################################################################
+name: 'Set milestone on issue of merged PR'
+
+on:
+  pull_request:
+    types:
+      - closed
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  set-milestone:
+    name: 'Set milestone on issue of merged PR #${{ github.event.pull_request.number }}'
```

Comment:
@romani Aren't these names not friendly for mobile? You are the one who is mobile heavy and wanted us to shorten other names.

Reply:
Yes, better to shorten

Reply:
Just "Set milestone"?

Reply:
I don't know how much space we have to work with but I would do `Milestone for PR #####`

Reply:
Done.